hhs_oauth_server/settings/base.py

```diff
@@ -500,17 +492,22 @@
     'social_core.pipeline.mail.mail_validation',
     'social_core.pipeline.user.create_user',
     'apps.accounts.auth_backends.pipeline.create_user_profile',
+    'apps.fhir.bluebutton.auth_backends.pso_pipeline.set_sample_patient_id',
```

Comment:
I may be missing something, but I don't see this system implemented as part of this PR. Won't this cause problems if merged in its current state?